cognee/api/v1/add/add.py

```
@@ -19,6 +22,73 @@
+
+
+async def _read_stream_bytes(stream: Any) -> bytes:
+    if not hasattr(stream, "read"):
+        raise TypeError(f"Expected stream-like object, got: {type(stream)}")
+
+    # Best effort to read from the start of the stream.
+    if hasattr(stream, "seek"):
+        try:
+            stream.seek(0)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)